packages/packages/commonwell-cert-runner/src/patient-management.ts

```diff
@@ -40,29 +42,48 @@ export async function patientManagement(
   console.log(`>>> D4a: Merge two Patient records`);
   // Create a second patient
   const respPatient2 = await commonWell.registerPatient(queryMeta, mergePatient);
-  const patientId2 = getIdPatient(respPatient2);
+  const patientId2 = getIdTrailingSlash(respPatient2);
   const referenceLink = respD1b._links.self.href;
 
   await commonWell.mergePatients(queryMeta, patientId2, referenceLink);
   console.log("D4a successful");
 
+  // LINK MANAGEMENT
+  // D5: Patient Matches
   console.log(`>>> D5a: Retrieve network links`);
   // Main Account Link
   const person = await commonWell.enrollPerson(queryMeta, personStrongId);
   const personId = getId(person);
   await commonWell.patientLink(queryMeta, personId, referenceLink);
   // Sandbox Account Link
-  patient.identifier[0].system = `urn:oid:${commonwellSandboxOID}`;
-  patient.identifier[0].assigner = commonwellSandboxOrgName;
-  patient.identifier[0].label = commonwellSandboxOrgName;
-  const sandboxPatient = await commonwellSandbox.registerPatient(queryMeta, patient);
-  const sandboxPatientId = getIdPatient(sandboxPatient);
+  let payloadSandboxPatient = cloneDeep(patient);
```

Comment:
Need to clone the object because in link management it was using the object that got updated and was breaking